jobs/generation/Utilities.groovy

```diff
@@ -26,18 +26,47 @@ class Utilities {
     return getFolderName("${project}_${branch}_${targetArch}_${config}")
   }
 
+  def private static String getDockerImage(String branch) {
+    switch (branch) {
+      case 'release/2.0.0':
+        return 'hqueue/dotnetcore:ubuntu1404_cross_prereqs_v4-tizen_rootfs'
+      case 'release/2.1':
+      case 'release/2.2':
+        return 'tizendotnet/dotnet-buildtools-prereqs:ubuntu-16.04-cross-e435274-20180426002255-tizen-rootfs-5.0m1'
+      case 'release/3.0':
+      case 'master':
+        return 'tizendotnet/dotnet-buildtools-prereqs:ubuntu-16.04-cross-10fcdcf-20190208200917-tizen-rootfs-5.0m2'
```

Comment:
Shouldn't this be 5.5m1?

Reply:
Yes, but firstly we need to add support for `tizen.5.5.0` rid.